src/test/java/com/puppycrawl/tools/checkstyle/filters/SuppressionXpathFilterTest.java

```diff
@@ -179,4 +180,85 @@ private static SuppressionXpathFilter createSuppressionXpathFilter(String fileNa
         return suppressionXpathFilter;
     }
 
+    private LocalizedMessage getMessageForStringLiteral(int lineNo, int columnNo) {
+        return new LocalizedMessage(lineNo, columnNo, TokenTypes.PLUS, "", "",
+                null, null, "777", getClass(), null);
+    }
+
+    private LocalizedMessage getMessageForCharLiteral(int lineNo, int columnNo) {
+        return new LocalizedMessage(lineNo, columnNo, TokenTypes.CHAR_LITERAL, "", "",
+                null, null, "777", getClass(), null);
+    }
```

Comment:
it is better now, but can be improved even more. There is no need for separate methods to create a message. You can inline it into TreeWalkerAudit creation method.

Method signature will be like
```
private static TreeWalkerAuditEvent getTreeWalkerAuditForString(int lineNo, int columnNo, TokenTypes tokenType, File file)
```

Reply:
Done